jquery.ui.layout/jquery.ui.layout.d.ts

```diff
@@ -1,40 +1,484 @@
 // Type definitions for jQuery UI Layout Plug-in
 // Project: http://layout.jquery-dev.net/
-// Definitions by: Steve Fenton <https://github.com/Steve-Fenton>
+// Definitions by: Douglas Armstrong <https://github.com/drarmstr>
```

Comment:
sorry, one more things. please 'append' your name below original author.
like this https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/jquery/jquery.d.ts#L3